File: src/desktop/Models/Entry.cs
#nullable enable

using System;

namespace Transleet.Desktop.Models;


public class Entry
{
    public Guid Id { get; set; }
    public string Text { get; set; }
    public Locale Locale { get; set; }

    public bool IsSource { get; set; }

    public bool IsSuggestion { get; set; }

    public Translation Translation { get; set; }
}

File: src/vue/src/lib/api/models/Entry.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { Locale } from './Locale';
import type { Translation } from './Translation';

export type Entry = {
    id?: string;
    text?: string | null;
    locale?: Locale;
    isSource?: boolean;
    isSuggestion?: boolean;
    translation?: Translation;
};
